README.md

```diff
@@ -272,52 +197,42 @@ and one indicating a stability "dev". For example:
 }
 ```
 
-## The "lock file” and the "package hash"
+## The "lock file" and the "package hash".
 
-Everytime the assets are processed for a package, a lock file named **`.composer_compiled_assets`**
-is saved in the package's root.
+Each time the assets for a package are processed, a lock file named **`.composer_compiled_assets`** is saved in the root directory of the package.
 
-That file should be git-ignored if the compiled assets are git-ignored.
+This file should be git-ignored if the compiled assets are git-ignored.
 
-The lock file contains an hash, referenced as the "*Composer Asset Compiler package's hash*", which
-is created from the Composer Assets Compiler configuration and the content of specific files
-like `package.json`, `package-lock.json`, `npm-shrinkwrap.json`, `yarn.lock`.
+The lock file contains a hash, referenced as "*Composer Asset Compiler package's hash*", which is created from the Composer Assets Compiler configuration and the contents of certain files such as `package.json`, `package-lock.json`, `npm-shrinkwrap.json`, and `yarn.lock`.
 
-When a lock file is found, and the content of the hash in it matches the current package's hash, the
-Composer Assets Compiler plugin will not process the package's assets.
+If a lock file is found and the content of the hash in it matches the hash of the current package, the Composer Assets Compiler plugin will not process the assets of the package.
 
-Considering the Composer Asset Compiler package's hash does *not* depend from the actual assets
-source file, it might be possible that Composer Assets Compiler skip processing a package's assets
-even if the assets source have changed.
+Since the hash of the Composer Assets Compiler package does *not* depend on the actual asset source file, the Composer Assets Compiler might skip processing the assets of a package even if the asset source has changed.
 
-To prevent that, it is possible to use the `--ignore-lock` flag when executing the compile assets
-command:
+To prevent this, it is possible to use the `--ignore-lock` flag when running the compile assets command command:
 
 ````shell
 composer compile-assets --ignore-lock
 ````
 
-The Composer Asset Compiler package's hash can also be calculated programmatically via the command:
+The hash of the composer asset compiler package can also be calculated programmatically using the command:
```

Comment:
I think this changes the meaning of the sentence.

The hash does not belong to Composer Asset Compiler.

A wording, but precise sentence would be: _"the hash that Composer Asset Compiler internally calculates for a package can be programmatically calculated using the command:"_ .

I would like to keep this meaning using a less wording sentence, but my feeling is that the sentence you wrote change the meaning. But maybe the wording sentence is acceptable.

Reply:
What about this?
```suggestion
The hash of the package can also be calculated programmatically using the command:
```

That the `assets-hash` command is part of the Composer Asset Compiler should be clear; no need to mention it explicitly.